opus/application/static_media/js/browse.js

```diff
@@ -642,6 +658,13 @@ var o_browse = {
                     "step": o_browse.gallerySliderStep,
                     "max": opus.resultCount,
                 });
+
+                // update obsNum in both infiniteScroll instances
+                // store the most top left obsNum in gallery for gallery infiniteScroll instance
+                // store the most top obsNum in table for table infiniteScroll instance
+                $(`${tab} .op-gallery-view`).infiniteScroll({"obsNum": obsNum});
+                $(`${tab} .op-dataTable-view`).infiniteScroll({"obsNum": obsNum});
```

Comment:
based on notes in review, i suspect this will change.
You just need to update the slider rounding down the obs number from the table. Whatever the JS equivalent is of
Floor(obsnum / rowsize) * rowsize (maybe adjusted for a 1 start instead of 0)